build-tools/create-packs/Directory.Build.targets

```diff
@@ -61,16 +61,31 @@
-    <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=32 -p:AndroidRID=android-x86 -p:AndroidABI=x86-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
-    <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=32 -p:AndroidRID=android-x64 -p:AndroidABI=x86_64-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
-    <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=32 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Ref.proj&quot;" />
+  <Target Name="_CreateOtherApiLevels">
+    <Exec
+        Condition=" '%(AndroidApiInfo.DotNet)' == 'true' "
+        Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=%(AndroidApiInfo.Level) -p:AndroidRID=android-arm -p:AndroidABI=armeabi-v7a-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;"
+    />
+    <Exec
+        Condition=" '%(AndroidApiInfo.DotNet)' == 'true' "
+        Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=%(AndroidApiInfo.Level) -p:AndroidRID=android-arm64 -p:AndroidABI=arm64-v8a-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;"
+    />
+    <Exec
+        Condition=" '%(AndroidApiInfo.DotNet)' == 'true' "
+        Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=%(AndroidApiInfo.Level) -p:AndroidRID=android-x86 -p:AndroidABI=x86-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;"
+    />
+    <Exec
+        Condition=" '%(AndroidApiInfo.DotNet)' == 'true' "
+        Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=%(AndroidApiInfo.Level) -p:AndroidRID=android-x64 -p:AndroidABI=x86_64-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;"
+    />
+    <Exec
+        Condition=" '%(AndroidApiInfo.DotNet)' == 'true' "
+        Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidApiLevel=%(AndroidApiInfo.Level) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Ref.proj&quot;"
+    />
   </Target>
 
   <Target Name="CreateAllPacks"
-      DependsOnTargets="DeleteExtractedWorkloadPacks;_SetGlobalProperties;GetXAVersionInfo;_CleanNuGetDirectory;_CreateAPI32Packs;_CreateDefaultRefPack">
+      DependsOnTargets="DeleteExtractedWorkloadPacks;_SetGlobalProperties;GetXAVersionInfo;_CleanNuGetDirectory;_CreateOtherApiLevels;_CreateDefaultRefPack">
     <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidRID=android-arm -p:AndroidABI=armeabi-v7a-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
     <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidRID=android-arm64 -p:AndroidABI=arm64-v8a-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
     <Exec Command="dotnet pack @(_GlobalProperties, ' ') -p:AndroidRID=android-x86 -p:AndroidABI=x86-net6 &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Runtime.proj&quot;" />
```

Comment:
I think these ref/runtime pack creation calls for the default API level are no longer needed with these changes. Maybe we can remove these and the `_CreateOtherApiLevels` target, and move the new Exec calls down here instead?

Reply:
I removed all the extra targets, so it batches each API level now for the runtime and ref packs.